packages/gatsby/src/utils/worker/child/queries.ts

```diff
@@ -3,16 +3,20 @@ import {
   runPageQueries,
   runStaticQueries,
 } from "../../../services"
-import { store } from "../../../redux"
+import { savePartialStateToDisk, store } from "../../../redux"
 import { GraphQLRunner } from "../../../query/graphql-runner"
 import { getDataStore } from "../../../datastore"
 import { setState } from "./state"
 import { buildSchema } from "./schema"
 
-export function setQueries(): void {
+export function setComponents(): void {
```

Comment:
Since I named the new function `saveQueries` this name was a bit conflicting thus needed to be renamed, too